utils.js

```diff
@@ -696,6 +696,15 @@ function getType(obj) {
   return Object.prototype.toString.call(obj).slice(8, -1);
 }
 
+function formatProxyPresence(presence, userID) {
+  if(presence.lat === undefined) return null;
+  return {
```

Comment:
As I written in my reply in pull request, the `type: presence` returns only ONE user presence, this return an list of all users and their presences. For make it match with the signature of default presence event, every user's presence in array will must call callback, what is not good idea.